subprojects/docs/src/samples/webApplication/customized/build.gradle

```diff
@@ -1,4 +1,13 @@
-apply plugin: 'jetty'
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'org.akhikhl.gretty:gretty:1.4.2'
```

Comment:
Never mind...maybe we'll just keep it in and fix it if it breaks. It's good to have a working example that people can directly play around with.